include/groonga/util.h

```diff
@@ -172,6 +172,46 @@ grn_inspect_limited(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj);
  */
 GRN_API grn_obj *
 grn_inspect_name(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj);
+/**
+ * \brief Inspect the given encoding and produce its name.
+ *
+ * \since 4.0.8
```

Comment:
```suggestion
 * \since 2.1.0
```

See: https://github.com/groonga/groonga/commit/1e407fef40a1d7287c0d00ef922bd1907077e876#diff-65b4226c3c4589432ae5d2fd3214ba6d01c0788ad3dd336fde0b20d1f0b911daR85

Reply:
This API was published at v4.0.8. So I think this version is better.
https://github.com/groonga/groonga/commit/4e92dba73a2587aa99dd31767492e7f0c3865204

Reply:
I see.